packages/panna-sdk/src/core/utils/activity.ts

```diff
@@ -329,9 +414,9 @@ export const getActivitiesByAddress = async function (
         const amountType = getAmountType(address, tx);
         switch (amountType) {
           case TokenERC.ETH: {
-            amount = {
+            const ethAmount: EtherAmount = {
               type: TokenERC.ETH,
-              value: tx.value,
+              value: tx.value || '0',
```

Comment:
We shouldn't default value to `0`. In case, there's no value returned that means the `amountType` determination logic is broken or doesn't cover some edgecase.

Reply:
Done.